### PR TITLE
docs: fix type in remix adapter

### PR DESCRIPTION
**Pull request by bravo-kernel**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,19988 @@
+{
```
Comment (airjp73) [inline package-lock.json:1]: Let's not commit this. I'm using bun for this project.

**Review by airjp73 — CHANGES_REQUESTED** [review]
Thanks! Just one tweak before we can merge.

File: package-lock.json (hunk 1 above)
Comment (bravo-kernel) [reply package-lock.json]: Apologies, totally missed that. Should be good now.

Comment (airjp73) [reply package-lock.json]: No worries!

**Review by airjp73 — APPROVED** [review]

**Pull request merged**